wrappers/javascript/indy-vdr-react-native/cpp/indyVdr.cpp

```diff
@@ -32,6 +32,16 @@ jsi::Value setCacheDirectory(jsi::Runtime &rt, jsi::Object options) {
   return createReturnValue(rt, code, nullptr);
 };
 
+jsi::Value setLedgerTxnCache(jsi::Runtime &rt, jsi::Object options) {
+  auto capacity = jsiToValue<size_t>(rt, options, "capacity");
+  auto expiry_offset_ms = jsiToValue<c_ulong>(rt, options, "expiry_offset_ms");
+  auto path = jsiToValue<std::string>(rt, options, "path");
```

Comment:
if path is optional the following can be used:

```suggestion
  auto path = jsiToValue<std::string>(rt, options, "path", true);
```

But make sure that before you call `path.c_str()` you first check whether its a nullptr or string. There should be some examples in other funtions to check it from.